### ISSUE TITLE
PropertyUserStoreTest failures in Windows

### ISSUE THREAD

**Issue by joakime** [eclipse/jetty.project#1789]
These are the results of building on Windows 10 using Java 8u144 against branch jetty-9.4.x-1200 (from Issue #1200)

``` shell
[ERROR] Tests run: 4, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 4.3 s <<< FAILURE! - in org.eclipse.jetty.security.PropertyUserStoreTest
[ERROR] testPropertyUserStoreLoadFromJarFile(org.eclipse.jetty.security.PropertyUserStoreTest)  Time elapsed: 0.021 s  <<< ERROR!
java.lang.IllegalStateException: java.lang.RuntimeException: java.net.URISyntaxException: Illegal character in opaque part at index 11: jar:file:C:\code\jetty\jetty.project-9.4.x\jetty-security\target\tests\testProper...reLoadFromJarFile\users.jar!/mountain_goat/pale_ale.txt
        at org.eclipse.jetty.security.PropertyUserStoreTest.testPropertyUserStoreLoadFromJarFile(PropertyUserStoreTest.java:197)
Caused by: java.lang.RuntimeException: java.net.URISyntaxException: Illegal character in opaque part at index 11: jar:file:C:\code\jetty\jetty.project-9.4.x\jetty-security\target\tests\testProper...reLoadFromJarFile\users.jar!/mountain_goat/pale_ale.txt
        at org.eclipse.jetty.security.PropertyUserStoreTest.testPropertyUserStoreLoadFromJarFile(PropertyUserStoreTest.java:197)
Caused by: java.net.URISyntaxException: Illegal character in opaque part at index 11: jar:file:C:\code\jetty\jetty.project-9.4.x\jetty-security\target\tests\testProper...reLoadFromJarFile\users.jar!/mountain_goat/pale_ale.txt
        at org.eclipse.jetty.security.PropertyUserStoreTest.testPropertyUserStoreLoadFromJarFile(PropertyUserStoreTest.java:197)

[ERROR] testPropertyUserStoreLoadUpdateUser(org.eclipse.jetty.security.PropertyUserStoreTest)  Time elapsed: 2.127 s  <<< ERROR!
java.lang.UnsupportedOperationException: 'posix:permissions' not supported as initial attribute
        at org.eclipse.jetty.security.PropertyUserStoreTest.testPropertyUserStoreLoadUpdateUser(PropertyUserStoreTest.java:245)
```

**Comment by gregw** [comment]
Fixed in the branch for #1200

**Comment by joakime** [comment]
Confirmed fixed.
